src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -121,5 +121,11 @@
             <argument type="service" id="request_stack" />
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="container.env_var_processor" class="Symfony\Component\DependencyInjection\EnvVarProcessor">
+            <tag name="container.env_var_processor" />
```

Comment:
shouldn't it have an heigher priority over secrets?

Reply:
I think you missed these are two different tags.